globalConfig.json

```diff
@@ -1,716 +1,779 @@
 {
-  "meta": {
-    "name": "TA_dataset",
-    "displayName": "Singularity Data Lake Add-On for Splunk",
+    "meta": {
```

Comment:
would it be possible to avoid reformating of the file? it difficult to review these changes

Reply:
i made only in the account configuration section only. not sure why its showing these many lines code change.

Reply:
I have checked the changes despite different spacing